samples/Sample.AzureServiceBus.InMemory/SampleSubscriber.cs

```diff
@@ -1,14 +1,23 @@
 using DotNetCore.CAP;
+using Sample.AzureServiceBus.InMemory.Contracts.DomainEvents;
+using Sample.AzureServiceBus.InMemory.Contracts.IntegrationEvents;
 
 namespace Sample.AzureServiceBus.InMemory;
 
 public class SampleSubscriber : ICapSubscribe
 {
     public record Message(string Content);
     
-    [CapSubscribe("cap.sample.tests")]
+    [CapSubscribe(nameof(EntityCreated))]
     public void Handle(Message message)
     {
         Console.WriteLine($"Message {message.Content} received");
     }
+    
+    [CapSubscribe(nameof(EntityCreatedForIntegration), Group = "test")]
+    [CapSubscribe(nameof(EntityDeleted))]
```

Comment:
Are we trying to have two different subscriptions for the same message here? Just to understand the two `CapSubscribe` attributes.

Reply:
Thx I just tried it for only diffrent message. For same are exetuce both of subscriber handlers.